ui/app/templates/vault/cluster/secrets/backends.hbs

```diff
@@ -59,6 +59,23 @@
                      View Configuration
                    {{/link-to}}
                   </li>
+                  {{#unless (eq backend.type "cubbyhole")}}
+                  <li class="action">
+                    <ConfirmAction
+                      @confirmButtonClasses="button is-primary"
+                      @buttonClasses="link is-destroy"
+                      @onConfirmAction={{perform disableEngine backend}}
+                      @confirmMessage={{concat "Are you sure you want to disable the " backend.engineType " engine at " backend.path "?"}}
+                      @showConfirm={{get this (concat "shouldDelete-" method.id)}}
+                      @class={{if (get this (concat "shouldDelete-" method.id)) "message is-block is-warning is-outline"}}
+                      @containerClasses="message-body is-block"
+                      @messageClasses="is-block"
+                      @confirmButtonText="Disable"
+                      data-test-engine-disable>
+                      Disable
+                    </ConfirmAction>
+                   </li>
+                   {{/unless}}
                  {{#if item.updatePath.isPending}}
                    <li class="action">
                      <button disabled=true type="button" class="link button is-loading is-transparent"></button>
```

Comment:
Hey! :wave:

Super nit: and not in this PR. Whilst having a scan of this I saw this `<button disabled=true>`.

Should that be `<button disabled>` or `<button disabled="disabled">` or `<button disabled="true">`, basically if all the other HTML attributes are with `"` then should this one be also.

John

Reply:
hey, thanks for the feedback @johncowen! i updated this along with the rest of the codebase to use quotes for consistency. other than making `disabled` consistent with other html attributes, are there benefits for this change? i don't have strong opinions here so want to make sure i understand better for next time!

Reply:
Hey! Superstar!

> are there benefits for this change?

Apart from consistency none at all (that I'm aware of), hence the 'super nit'. I suppose I saw it and made me wonder for a split second whether this was something in js/ember-world, made me double take, and then I thought no it can't be that `<button>` has a lowercase `b`.

I did have a quick look and looks like HTML5 doesn't care if you use `"` or not so 🤷‍♂️

Reply:
awesome. thanks for clarifying! nits are always welcome. :)